test/test_non_number_eval_tree_array.jl

```diff
@@ -0,0 +1,72 @@
+
+using DynamicExpressions
+using DynamicExpressions: Node, @extend_operators, OperatorEnum
+
+# SVM (scalar, vector, matrix) - struct that contains all three datatypes
+struct SVM{T}
+    dims::Int8 # number of dimmentions
```

Comment:
```suggestion
    dims::UInt8 # number of dimmentions
```
Not a big deal though